packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -265,6 +265,49 @@ void main() {
     return completer.future;
   }
 
+  testWidgets('Set description while recording', (WidgetTester tester) async {
+    final List<CameraDescription> cameras = await availableCameras();
+    if (cameras.length < 2) {
+      return;
+    }
+
+    final CameraController controller = CameraController(
+      cameras[0],
+      ResolutionPreset.low,
+      enableAudio: false,
+    );
+
+    await controller.initialize();
+    await controller.prepareForVideoRecording();
+
+    await controller.startVideoRecording();
+    sleep(const Duration(milliseconds: 500));
```

Comment:
Carrying comments over from implementation tests: do we really need all these half-second sleeps? If so, why?